src/gluonts/nursery/duru/data.py

```diff
@@ -0,0 +1,488 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+import gluonts.dataset
+import torch
+import torch as T
+import numpy as np
+import sklearn.model_selection
+import pickle
+import os
+import torchvision.transforms as transforms
+from gluonts.dataset.repository.datasets import get_dataset, dataset_recipes
+from gluonts.dataset.split import OffsetSplitter
+from utils import calculate_padding_base_2
```

Comment:
This should be a relative import, `from .utils import ...`.

Reply:
It works I guess, given that the project folder is the repository. Do we need to change it?

Reply:
Yes!

This won't work if it's used as a package.